src/open_samus_returns_rando/pickups/pickup.py

```diff
@@ -14,10 +14,13 @@
 from open_samus_returns_rando.patcher_editor import PatcherEditor, path_for_level
 from open_samus_returns_rando.pickups.model_data import get_data
 
-RESERVE_TANK_ITEMS = {
+RESERVE_TANKS = {
     "ITEM_RESERVE_TANK_LIFE",
     "ITEM_RESERVE_TANK_MISSILE",
     "ITEM_RESERVE_TANK_SPECIAL_ENERGY",
+    "powerup_reserveenergy",
+    "powerup_reservemissile",
+    "powerup_reserveaeion",
```

Comment:
Oh wait: Doesn't these new values need entries in the `MODEL_DATA` thing for the bcmdl path and dependencies etc.?